src/ui/components/InputRequest/InputRequest.tsx

```diff
@@ -64,6 +66,13 @@ const InputRequest = () => {
   }, [showModal])
 
   const resolveConnectionOobi = async (content: string) => {
+    if (
+      !new URL(content).pathname.match(OOBI_AGENT_ONLY_RE) &&
+      !new URL(content).pathname.match(WOOBI_RE)
+    ) {
+      throw new Error("Invalid connection url");
```

Comment:
Hmm, @Sotatek-DukeVu what is the best way to do this in the UI? Can you contribute to this PR

Reply:
I at least see the UI has a proper ErrorMessage.INVALID_CONNECTION_URL

Reply:
I don't think we need validate connection value in here.
`resolveConnectionOobi` called when user had scanned a connection that's missing connection name. This mean connection has been validated on scanner.

Reply:
OK, so we can revert this. Do the other changes look OK?